grl/environment/policy_lib.py

```diff
@@ -36,11 +36,11 @@ def tiger_alt_start_cam(**kwargs) -> jnp.ndarray:
     pi_params = reverse_softmax(pi_phi)
     return pi_params
 
-def get_start_pi(pi_name: str, spec: dict = None, **kwargs):
+def get_start_pi(pi_name: str, pi_phi: jnp.ndarray = None, **kwargs):
     # if we pass in a digit for get_start_pi, that means we look into
     # the corresponding index in Pi_phi in spec.
```

Comment:
This comment doesn't make sense to me. Is it still correct?

Reply:
deleted!